react/features/base/conference/middleware.any.ts

```diff
@@ -386,6 +386,8 @@ function _connectionFailed({ dispatch, getState }: IStore, next: Function, actio
     const { connection, error } = action;
     const { jwt } = getState()['features/base/jwt'];
 
+    dispatch(stopLocalVideoRecording());
+
```

Comment:
Shouldn't this be on conferenceFailed not connectionFailed? Just before the reload?
We also have a check in the same middleware whether recording is ongoing and do it only then:
https://github.com/jitsi/jitsi-meet/blob/9b798b4514da08ef027e336d3a3abcff35c38faf/react/features/base/conference/middleware.any.ts#L288